src/pages/notifications/index.vue

```diff
@@ -206,7 +206,11 @@ export default {
       'getNFTClassMetadataById',
       'getNotificationCount',
       'getEventLastSeenTs',
+      'getLatestEventTimestamp',
     ]),
+    latestEventTimestamp() {
+      return this.getLatestEventTimestamp;
+    },
```

Comment:
Why computed??

Reply:
I thought the **immediate: true** property would only start watching after the **created** lifecycle hook, so I was worried that if `getLatestEventTimestamp` already has a value before entering the page, the handler wouldn't be triggered. That's why I wrapped it in a computed property.

Reply:
But later I found that if immediate: true, the handler will be triggered immediately without going through a process of going from undefined to having a value. 😝